<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Особенности пациента</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="frmWidgets" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="2" column="7">
       <widget class="QLabel" name="lblMenarhe">
        <property name="text">
         <string>Менархе</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="lblMenoPausa">
        <property name="text">
         <string>Менопауза</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QSpinBox" name="edtMenarhe"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblBloodType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Группа крови</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CRBComboBox" name="cmbBloodType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblBloodTypeDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Дата установления</string>
        </property>
       </widget>
      </item>
      <item row="2" column="11">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="CDateEdit" name="edtBloodTypeDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBirthHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Рост при рождении</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="edtBirthHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>в сантиметрах</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lblBirthWeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Вес при рождении</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="edtBirthWeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>в граммах</string>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSpinBox" name="edtBirthGestationalAge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>в неделях</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="12">
       <widget class="QGroupBox" name="grbAllergy_2">
        <property name="title">
         <string>Медикаментозная непереносимость</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="margin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="CInDocTableView" name="tblIntoleranceMedicament"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="6" colspan="6">
       <widget class="QLineEdit" name="edtBloodTypeNotes"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="lblBirthGestationalAge">
        <property name="text">
         <string>Срок рождения</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QLabel" name="lblBloodTypeNotes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Примечания</string>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QSpinBox" name="edtMenoPausa"/>
      </item>
      <item row="3" column="0" colspan="12">
       <widget class="QGroupBox" name="grbAllergy">
        <property name="title">
         <string>Аллергия</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="CInDocTableView" name="tblAllergy"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="12">
       <widget class="QTextBrowser" name="txtClientInfoBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>frmWidgets</tabstop>
  <tabstop>cmbBloodType</tabstop>
  <tabstop>edtBloodTypeDate</tabstop>
  <tabstop>edtBloodTypeNotes</tabstop>
  <tabstop>edtBirthHeight</tabstop>
  <tabstop>edtBirthWeight</tabstop>
  <tabstop>edtBirthGestationalAge</tabstop>
  <tabstop>edtMenarhe</tabstop>
  <tabstop>edtMenoPausa</tabstop>
  <tabstop>tblAllergy</tabstop>
  <tabstop>tblIntoleranceMedicament</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>807</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>807</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
